ext/standard/php_var.h

```diff
@@ -48,6 +48,8 @@ struct php_unserialize_data {
 	void *last;
 	void *first_dtor;
 	void *last_dtor;
+	void *first_wakeup;
```

Comment:
You are right, this change would be an ABI break. DSOs are expected to work between patch versions without recompilation. (And the allocation of the structure is part of a macro, so it will be wrong even if extensions don't manually use the structure.)